src/shogun/base/SGObject.cpp

```diff
@@ -650,17 +646,17 @@ void CSGObject::build_gradient_parameter_dictionary(CMap<TParameter*, CSGObject*
 
 bool CSGObject::equals(CSGObject* other, float64_t accuracy, bool tolerant)
 {
-	SG_DEBUG("entering %s::equals()\n", get_name());
+	SG_DEBUG("entering equals()\n");
```

Comment:
If you are keen, you could fix the punctuation on such messages: 
`SG_DEBUG("Entering.\n");`